src/Uno.UWP/Storage/Pickers/FileSavePicker.macOS.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+using Windows.Foundation;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FileSavePicker
+	{
+		public PickerLocationId SuggestedStartLocation { get; set; }
+		public IAsyncOperation<StorageFile> PickSaveFileAsync() => PickFileTaskAsync().AsAsyncOperation();
+
+
+		private async Task<StorageFile> PickFileTaskAsync()
+		{
+			var savePicker = new NSSavePanel();
+			savePicker.DirectoryUrl = new NSUrl(GetStartPath(), true);
+			savePicker.AllowedFileTypes = GetFileTypes();
+			if(SuggestedFileName != null)
```

Comment:
nit: missing space after `if`

Reply:
I THINK it is. I just found documentation for it, but it was my fault for not executing a Format Document on everything before committing.

Reply:
No worries at all, coding guidelines evolved a bit over the course of time, though reformatting a whole document when modifying an existing one is generally not a good idea for a feature PR as it makes reviewing more complicated. For new files, it's a good idea! (Along with `#nullable enable`)